iree/compiler/Conversion/HLOToLinalg/HLOToLinalgOnBuffers.cpp

```diff
@@ -1550,9 +1557,8 @@ void ConvertHLOToLinalgOnBuffersPass::runOnFunction() {
   OutputBufferMap outputBufferMap;
   TensorToBufferMap resultTensorToBufferMap;
   if (failed(createAndPropagateBufferUsedForResultTensors(
-          funcOp, outputBufferMap, resultTensorToBufferMap))) {
+          funcOp, outputBufferMap, resultTensorToBufferMap)))
     return signalPassFailure();
-  }
```

Comment:
nit: we are trying not to do this in IREE code and use braces like in google style